File: app.Tabaldi.PACT.LibraryModels/AuthenticationModule/Models/ProfileModel.cs
using System;

namespace app.Tabaldi.PACT.LibraryModels.AuthenticationModule.Models
{
    public class ProfileModel
    {
        public int ID { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public string FullName { get; set; }
        public string Mail { get; set; }
        public bool SendAlerts { get; set; }
        public DateTimeOffset RegistrationDate { get; set; }
    }
}

File: Mobile/FisioTech/app/src/main/java/com/tabaldi/fisiotech/profile/ProfileModel.java
package com.tabaldi.fisiotech.profile;

import java.io.Serializable;

public class ProfileModel implements Serializable {
    public int id;
    public String userName;
    public String password;
    public String fullName;
    public String mail;
    public boolean sendAlerts;
    public String registrationDate;
}
